### ISSUE TITLE
SSE3 Build for X86 32bit systems?

### ISSUE THREAD

**Issue by gaining** [Alex313031/thorium#659]
**Is your feature request related to a problem? Please describe.**
No 32bit support for linux sse3 deb format?

**Describe the solution you'd like, including relevant patches or source**
I tried to install the sse3 deb build on a netbook with an Intel Atom® Processor N470 but it was only for amd64 cpus. Wondering if you can provide a build for 32 bit systems since you have one for Windows XP, most XP systems were 32 bit.

**Comment by gz83** [comment]
Now many new versions of Linux distributions are abandoning 32-bit software, and 32-bit may have a hard time in the future.

**Comment by gaining** [comment]
Yeah but I'm using peppermint OS based on debian 32bit arch on this netbook. It is designed for old computers like mine and kept up to date even with the latest linux kernel

**Comment by gaining** [comment]
Tried making a 32bit linux build but it seems that I'm stopped by this:
Generating files...
ERROR at //BUILD.gn:1632:1: Assertion failed.
assert(
^-----
'target_cpu=x86' is not supported for 'target_os=linux'. Consider omitting 'target_cpu' (default) or using 'target_cpu=x64' instead.
See //BUILD.gn:1633:5: 
    is_valid_x86_target || target_cpu != "x86" || v8_target_cpu == "arm",
    ^-------------------------------------------------------------------
This is where it was set.

**Comment by gz83** [comment]
It seems that Google has made an assertion at the source code level, prohibiting x86 builds for linux, and now there are no instances for linux x86 in Google's CI/CQ

@gaining

**Comment by Alex313031** [comment]
@gaining Building Thorium/Chromium for 32 bit Linux requires some patches, but it is possible (Debian 32 bit does it). I have made an SSE2 32 bit Linux build of Thorium 119 before, for running on a server I have.

I can make you a special 32 bit build.

**Comment by Alex313031** [comment]
@gaining Here you go > https://github.com/Alex313031/thorium/releases/tag/M123.0.6312.134
